Tests/backend-specific-tests/Synapse/RunnerSynapseTest.php

```diff
@@ -151,4 +163,102 @@ public function testWorkspaceMapping()
+        $csv->writeRow(['id', 'text']);
+        $csv->writeRow(['test1', 'test1']);
+        $this->getClient()->createTableAsync('in.c-synapse-runner-test', 'mytable', $csv);
+        $this->getClient()->uploadFile(
+            $csv->getPath(),
+            (new FileUploadOptions())
+                ->setTags([self::ABS_TEST_FILE_TAG])
+                ->setFileName('abs-workspace-file.csv')
+        );
+        unset($csv);
+
+        $componentData = [
+            'id' => 'keboola.runner-workspace-abs-test',
+            'data' => [
+                'definition' => [
+                    'type' => 'aws-ecr',
+                    'uri' => '061240556736.dkr.ecr.us-east-1.amazonaws.com/keboola.runner-workspace-test',
+                    'tag' => 'latest',
+                ],
+                'staging_storage' => [
+                    'input' => 'workspace-abs',
+                    'output' => 'workspace-abs',
+                ],
+            ],
+            // https://keboola.slack.com/archives/C02C3GZUS/p1598942156005100
+            // https://github.com/microsoft/msphpsql/issues/400#issuecomment-481722255
+            'features' => ['container-root-user'],
+        ];
+
+        $configId = uniqid('runner-test-');
+        $components = new Components($this->client);
+        $configuration = new Configuration();
+        $configuration->setComponentId('keboola.runner-workspace-abs-test');
+        $configuration->setName('runner-tests');
+        $configuration->setConfigurationId($configId);
+        $components->addConfiguration($configuration);
+        $runner = $this->getRunner();
+
+        self::assertFalse($this->client->tableExists('out.c-synapse-runner-test.new-table'));
+        $runner->run(
+            $this->prepareJobDefinitions(
+                $componentData,
+                $configId,
+                [
+                    'storage' => [
+                        'input' => [
+                            'tables' => [
+                                [
+                                    'source' => 'in.c-synapse-runner-test.mytable',
+                                    'destination' => 'local-table',
+                                ],
+                            ],
+                            'files' => [
+                                [
+                                    'tags' => [self::ABS_TEST_FILE_TAG],
+                                ],
+                            ],
+                        ],
+                        'output' => [
+                            'tables' => [
+                                [
+                                    'source' => 'local-table-out',
+                                    'destination' => 'out.c-synapse-runner-test.new-table',
+                                ],
+                            ],
+                        ],
+                    ],
+                    'parameters' => [
+                        'operation' => 'copy-abs',
```

Comment:
i think it has something like `list` to check if the input files are there

Reply:
:-( it doesn't, but apparently it doesn't even have `copy-abs`, I was sure I added it. https://github.com/keboola/runner-workspace-test/blob/master/src/ConfigDefinition.php#L21

Reply:
I will add the list operation there